scripts/yarn/export.js

```diff
@@ -1,3 +1,17 @@
 import { processEnv } from 'env-loader'
+import fs from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+// Clean the export directory to get tugboat working
+if (process.env.BUILD_OPTION === 'static') {
+  const __filename = fileURLToPath(import.meta.url)
+  const __dirname = path.dirname(__filename)
+
+  const exportBuildPath = path.resolve(__dirname, '../../.next/export')
+  if (fs.existsSync(exportBuildPath)) {
+    fs.rmdirSync(exportBuildPath, { recursive: true, force: true })
+  }
+}
```

Comment:
I can pull this out into a separate PR if'n y'like. I was just using this one as a test to see if it worked.

TL;DR: Tugboat was failing for reasons I don't actually understand, but this fixes it. I tried to hunt down _why_ NextJS was attempting to remove this directory, [but couldn't figure it out](https://dsva.slack.com/archives/C01SR56755H/p1741648853182619). Figured I'd try just removing the directory  myself before running the build and see if it did the trick. Turns out, it did. 🤷